### PR TITLE
Add translation domains to Adv. param controllers 2

### PR DESCRIPTION
**Pull request by AlexEven**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | develop.
| Description?  | Adding translation domains.
| Type?         | improvement
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | NA
| How to test?  | Check translation catalog export.

### CODE REVIEW

File: controllers/admin/AdminEmployeesController.php [hunk 1]
```diff
@@ -228,35 +234,44 @@ public function renderForm()
...
                     'name' => 'employee_avatar',
                     'html_content' => '<div id="employee-thumbnail"><a href="http://www.prestashop.com/forums/index.php?app=core&amp;module=usercp" target="_blank" style="background-image:url('.$obj->getImage().')"></a></div>
-					<div class="alert alert-info">'.sprintf($this->l('Your avatar in PrestaShop 1.6.x is your profile picture on %1$s. To change your avatar, log in to PrestaShop.com with your email %2$s and follow the on-screen instructions.'), '<a href="http://www.prestashop.com/forums/index.php?app=core&amp;module=usercp" class="alert-link" target="_blank">PrestaShop.com</a>', $obj->email).'</div>',
+					<div class="alert alert-info">'.sprintf($this->trans(
+                        'Your avatar in PrestaShop 1.6.x is your profile picture on %url%. To change your avatar, log in to PrestaShop.com with your email %email% and follow the on-screen instructions.',
+                        array(
+                            '%url%' => '<a href="http://www.prestashop.com/forums/index.php?app=core&amp;module=usercp" class="alert-link" target="_blank">PrestaShop.com</a>',
+                            '%email%' => $obj->email
```
Comment (mickaelandrieu) [inline controllers/admin/AdminEmployeesController.php]: comma is missing :)

```diff
- '%email%' => $obj->email
+ '%email%' => $obj->email,
```

File: controllers/admin/AdminEmployeesController.php [hunk 2]
```diff
@@ -228,35 +234,44 @@ public function renderForm()
...
+
+                        ),
+                        'Admin.AdvParameters.Help'
```
Comment (mickaelandrieu) [inline controllers/admin/AdminEmployeesController.php]: Same here

File: controllers/admin/AdminEmployeesController.php [hunk 3]
```diff
@@ -370,7 +385,7 @@ public function renderForm()
                     'name' => 'name',
                     'default' => array(
                         'value' => '',
-                        'label' => $this->l('-- Choose --')
+                        'label' => $this->trans('-- Choose --', array(), 'AdvParameters.Help')
```
Comment (mickaelandrieu) [inline controllers/admin/AdminEmployeesController.php]: Same here (for every item of an array, you need to finish by a comma)

File: controllers/admin/AdminRequestSqlController.php [hunk 4]
```diff
@@ -49,16 +49,16 @@ public function __construct()
 
         $this->fields_list = array(
             'id_request_sql' => array('title' => $this->trans('ID', array(), 'Admin.Global'), 'class' => 'fixed-width-xs'),
-            'name' => array('title' => $this->l('SQL query Name')),
-            'sql' => array('title' => $this->l('SQL query'))
+            'name' => array('title' => $this->trans('SQL query Name', array(), 'Admin.AdvParameters.Feature')),
+            'sql' => array('title' => $this->trans('SQL query', array(), 'Admin.AdvParameters.Feature'))
```
Comment (mickaelandrieu) [inline controllers/admin/AdminRequestSqlController.php]:
```diff
- 'Admin.AdvParameters.Feature'))
+ 'Admin.AdvParameters.Feature')),
```

**Review by mickaelandrieu — CHANGES_REQUESTED** [review]
Some little typos to address, good job @AlexEven :)

**Comment by AlexEven** [comment]
Thanks for the review @mickaelandrieu!

**Review by mickaelandrieu — APPROVED** [review]

**Comment by aleeks** [comment]
Thank you @AlexEven :bouquet:

**Pull request merged**